UI/ChatSignalR/UnoChat.Client/UnoChat.Shared/ViewModel.cs

```diff
@@ -220,6 +221,18 @@ private IDisposable ShouldSendModelsAddedToAllMessagesToMessageObserver(IObserve
                 .Delay(TimeSpan.FromMilliseconds(10), Schedulers.Default) // Wait for the list view to have been updated
```

Comment:
It's probably more readable to avoid duplicating the whole query, and just make the line optional.

```suggestion

#if !__WASM__
                .Delay(TimeSpan.FromMilliseconds(10), Schedulers.Default) // Wait for the list view to have been updated
#endif
```